samples/projects/quantum_backend/qsharp.json

```diff
@@ -0,0 +1,4 @@
+{
+    "files" : [ "src/GateSet.qs" ],
+    "packageType": "lib"
```

Comment:
nitpick: the `packageType` is no longer needed.

Reply:
Removed.